### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by shardulc**
@sushain97

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+[Apertium][1] is a free/open-source platform for developing rule-based machine
 translation systems.
 
-This repository holds submodules to core tools, other tools and all Apertium
+This repository holds submodules to core tools, other tools, and all Apertium
 language modules and pairs.
 
+To clone this repository with all its submodules, run
+
+    git clone --recurse-submodules --shallow-submodules --depth 1 https://github.com/apertium/apertium-all.git
```
Comment (sushain97) [inline README.md]: Suggest SSH URL.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
+
+(Change `--depth 1` to `--depth N` to get the last N commits.)
+
+To make a commit:
```
Comment (sushain97) [inline README.md]: Suggest: 

> To make a commit [add: "to each changed submodule"]

File: README.md [hunk 3]
```diff
@@ -1,9 +1,24 @@
...
+
+    git submodule foreach 'git commit -a -m <message>'
+    git submodule foreach 'git push'
+
+More information about using git is [on the wiki][2]. More information about the
```
Comment (sushain97) [inline README.md]: Capitalize Git :)

**Comment by sushain97** [comment]
We should probably also note that you should not be directly comitting to this repo and that it auto populates with repos with the topic `apertium-all` ?

**Comment by shardulc** [comment]
@sushain97, ping about this (I'm back from vacation :P)

**Review by sushain97 — APPROVED** [review]

**Pull request merged**